### DISCUSSION TITLE
What are disadvantages of using ping?

### DISCUSSION THREAD

**Discussion by Houdini** [ImageMagick/ImageMagick#5949] [General] [answered]
Hello,
task is get image resolution, we can do this with two different commands:

`magick identify -ping -format '%P' /tmp/image.png`
`magick identify -format '%P' /tmp/image.png`

ping is much faster as it doesn't read pixels. But does it have any disadvantages? Like it might gives wrong results sometimes, etc.

Thanks

**Comment by snibgo** [comment, marked as answer]
It shouldn't give wrong results. If it does, please report it.
